src/components/ConversationIcon.vue

```diff
@@ -65,6 +65,19 @@ import NcAvatar from '@nextcloud/vue/dist/Components/NcAvatar.js'
 
 import { CONVERSATION } from '../constants.js'
 
+const ICON_CLASS_BY_OBJECT_TYPE = {
+	file: 'icon-file',
+	'share:password': 'icon-password',
+	emails: 'icon-mail',
+}
+
+const ICON_CLASS_BY_TYPE = {
+	[CONVERSATION.TYPE.CHANGELOG]: 'icon-changelog',
+	[CONVERSATION.TYPE.ONE_TO_ONE_FORMER]: 'icon-user',
+	[CONVERSATION.TYPE.GROUP]: 'icon-contacts',
+	[CONVERSATION.TYPE.PUBLIC]: 'icon-public',
+}
```

Comment:
😊

Reply:
As mentioned in several of the backend PRs I don't think we want this.
All rooms should just return their avatar properly on the avatar endpoint, no matter if it is a custom uploaded one or a predefined fallback icon. This way the life of clients and frontend should be much easier as they simply use the avatar endpoint all the time, no more long list of if-switches to find the right class.

So drop this fallback behaviour from here, and then we fix the avatar endpoint to return proper icons in all the cases?